Jenkinsfile.nightly

```diff
@@ -2,7 +2,7 @@
 
 pipeline {
     agent {
-        label 'fixed'
+        label 'eloy'
```

Comment:
Is this for testing? It might be safer to have it on a separate commit that can be reverted, and maybe a note on the PR saying don't merge with this commit in the PR.